### PR TITLE
MES-5342 CPC Office Page Pass

### PR DESCRIPTION
**Pull request by tomcrawleyy**
## Description
Introduces the logic for the office page pass journey and fixes activity code which should have gone in with terminate journey.
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [x] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
image
image

### CODE REVIEW

**Review by andrewsetterfield — APPROVED** [review]

File: src/pages/office/cat-cpc/office.cat-cpc.page.html [hunk 1]
```diff
@@ -69,7 +82,7 @@ <h4>Test summary</h4>
             </candidate-description>
 
             <identification
-              *ngIf="!(isDelegated && isFail())"
+              *ngIf="!(isDelegated && isPass())"
```
Comment (szabikr) [inline src/pages/office/cat-cpc/office.cat-cpc.page.html]: These `*ngIf`s are getting pretty complex

File: src/pages/office/cat-cpc/office.cat-cpc.page.ts [hunk 2]
```diff
@@ -341,6 +364,16 @@ export class OfficeCatCPCPage extends BasePageComponent {
     this.store$.dispatch(new AdditionalInformationChanged(additionalInformation));
   }
 
+  activityCodeChanged(activityCodeModel: ActivityCodeModel) {
+    const showMeQuestion = this.form.controls['showMeQuestion'];
+    if (showMeQuestion) {
+      if (showMeQuestion.value && showMeQuestion.value.code === 'N/A') {
```
Comment (szabikr) [inline src/pages/office/cat-cpc/office.cat-cpc.page.ts]: could use the `get` function from `lodash` library to avoid the complexity of these if statements

**Review by szabikr — APPROVED** [review]

File: src/pages/office/cat-cpc/office.cat-cpc.page.html (hunk 1 above)
Comment (andrewsetterfield) [reply src/pages/office/cat-cpc/office.cat-cpc.page.html]: for idenification, isnt it as simple as if not a delegated test then display? im only looking at the UX not the story detail tho.

Comment (tomcrawleyy) [reply src/pages/office/cat-cpc/office.cat-cpc.page.html]: nice spot andy, addressed, not sure how i could amend the others to be more concise, open to suggestions

File: src/pages/office/cat-cpc/office.cat-cpc.page.ts (hunk 2 above)
Comment (tomcrawleyy) [reply src/pages/office/cat-cpc/office.cat-cpc.page.ts]: addressed, testing it manually showed no obvious change in behaviour and unit tests pass as normal regarding it so can assume working as expected

**Review by szabikr — APPROVED** [review]

File: src/pages/office/cat-cpc/office.cat-cpc.page.ts (hunk 2 above)
Comment (szabikr) [reply src/pages/office/cat-cpc/office.cat-cpc.page.ts]: Yep, looks good!

**Review by andrewsetterfield — APPROVED** [review]

**Pull request merged**